invokeai/app/services/model_install/model_install_default.py

```diff
@@ -501,7 +501,7 @@ def _scan_for_missing_models(self) -> list[AnyModelConfig]:
         """Scan the models directory for missing models and return a list of them."""
         missing_models: list[AnyModelConfig] = []
         for x in self.record_store.all_models():
-            if not Path(x.path).resolve().exists():
+            if not (self._app_config.models_path / x.path).resolve().exists():
```

Comment:
Why do we need to change this if the model paths are all absolute?

Reply:
At least on my system, there are still lots of model paths that are relative and the system complains horribly about missing models at startup time.